axoverlay/app/Makefile

```diff
@@ -17,4 +17,4 @@ $(PROG1): $(OBJS1)
 	$(STRIP) $@
 
 clean:
-	rm -f $(PROGS) *.o core *.eap
+	rm -f $(PROGS) *.o *.eap* *_LICENSE.txt package.conf* param.conf tmp*
```

Comment:
Maybe nitpick but do we need the last asterisk in *.eap* ?

Reply:
Yes, running acap-build two times will create a .eap.old.